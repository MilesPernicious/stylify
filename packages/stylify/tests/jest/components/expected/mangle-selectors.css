.a{
	padding: 8px
}
.a{
	background-color: #000
}
.a{
	display: inline-block
}
.a,
.c{
	font-size: 24px
}
.b{
	max-width: 800px
}
.b{
	margin: 0 auto
}
.c{
	color: green
}

@media (min-width: 768px) {
.c{
	font-size: 32px
}
}
