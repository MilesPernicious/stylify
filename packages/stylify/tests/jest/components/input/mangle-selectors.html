<button class="button"></button>
<div class="container"></div>
<h1 class="title"></h1>
