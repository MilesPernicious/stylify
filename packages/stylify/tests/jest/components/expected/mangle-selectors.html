<button class="a"></button>
<div class="b"></div>
<h1 class="c"></h1>
